constants.py

```diff
@@ -39,10 +41,12 @@ def _resource_path(relative_path: Path | str) -> Path:
 
     Works for dev and for PyInstaller.
     """
-    if IS_PACKAGED:
+    if IS_PACKAGED and not IS_APPIMAGE:
```

Comment:
```suggestion
    if IS_APPIMAGE:
        base_path = Path(sys.argv[0]).absolute().parent
    elif IS_PACKAGED:
```

This is where the "reversal" of paths should be happening. Check is moved here to simplify checking logic. If `sys.argv[0]` points to the `main.py` in the temporary folder, the parent folder should be used as the resource folder.